src/Commands/Plugin/InfoCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Plugin;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Plugins\PluginDiscovery;
+
+/**
+ * Class InfoCommand
+ * @package Pantheon\Terminus\Commands\Plugin
+ */
+class InfoCommand extends TerminusCommand
+{
+    /**
+     * Displays info about currently installed plugins.
```

Comment:
Can this be changed to specify Terminus plugins?

Reply:
Updated the comment in the latest commit